operator/internal/controller/redpanda/redpanda_controller.go

```diff
@@ -906,3 +907,7 @@ func allListTypes(c client.Client) ([]client.ObjectList, error) {
 	}
 	return types, nil
 }
+
+func IsFluxEnabled(useFlux *bool, forceDefluxedMode bool) bool {
```

Comment:
This might be better suited on either the reconciler itself or the ChartRef as you can eliminate one of the parameters that way. This is just an alias to `ptr.Deref` which isn't any more simple to think about.

```go
// Consider the following: The parameters are clear because they're typed and the total invocation is shorter.
r.isFluxEnabled(rp)

// Less to type but the parameters are less clear.
rp.IsFluxEnabled(r.ForceDefluxedMode)
```

Reply:
That function was not created for easier reading, but rather to have unit test.

Reply:
You can also unittest a method :D

Reply:
True, if that's what you want I'm happy to change it.